src/systems/wheel_slip/WheelSlip.cc

```diff
@@ -248,29 +255,41 @@ void WheelSlipPrivate::Update(EntityComponentManager &_ecm)
   for (auto &linkSurface : this->mapLinkSurfaceParams)
   {
     auto &params = linkSurface.second;
-    const auto * wheelSlipCmdComp =
-      _ecm.Component<components::WheelSlipCmd>(linkSurface.first);
```

Comment:
Deleting this line removes the ability to set wheel slip parameters via the ECM. I think it would be better to keep the ECM support (since it is more deterministic) but to add the gz-transport parameters interface in addition to it